src/components/CardSection.js

```diff
@@ -0,0 +1,34 @@
+import searchIcon from "../assets/Search.svg";
+import CardList from "./CardList";
+import "./CardSection.css";
+
+export default function CardSection({ cardList }) {
+  console.log(cardList);
+  return (
+    <div className="CardSection">
+      <div className="cardFrame">
+        <div className="searchBar">
+          <div className="searchBarElement">
+            <img id="searchIcon" src={searchIcon} alt="검색 아이콘"></img>
+            <input id="inputText" placeholder="링크를 검색해보세요."></input>
```

Comment:
input 태그는 type 속성을 넣어주지 않으면 type='text'로 적용되긴 하지만 명시적으로 넣어주시는게 좋아요~
```suggestion
            <img id="searchIcon" src={searchIcon} alt="검색 아이콘"></img>
            <input type='text' id="inputText" placeholder="링크를 검색해보세요."></input>
```